### PR TITLE
Implemented cypress tests for acknowledging alerts.

### PR DESCRIPTION
**Pull request by AWSHurneyt**
### Description
Implemented cypress tests for acknowledging alerts.

### Issues Resolved
https://github.com/opensearch-project/security-analytics-dashboards-plugin/issues/104

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: cypress/integration/3_alerts.spec.js [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+
+    // Confirm flyout has been closed
+    cy.contains('[data-test-subj="alert-details-flyout"]').should('not.exist');
+  });
+
+  it('contain expected values in finding details flyout', () => {
+    // Open first alert details flyout
+    cy.get('tbody > tr', TWENTY_SECONDS_TIMEOUT)
+      .first()
+      .within(() => {
```
Comment (phestecAMZN) [inline cypress/integration/3_alerts.spec.js:188]: Are we duplicating the test first alert flyout?

**Comment by phestecAMZN** [comment]
Looks like you got the same error I was getting -```
 FATAL  Error: Failed to initialize plugins:
	Plugin "securityAnalyticsDashboards" is only compatible with OpenSearch Dashboards version "2.4.0", but used OpenSearch Dashboards version is "2.4.1". (incompatible-version, /home/runner/work/security-analytics-dashboards-plugin/security-analytics-dashboards-plugin/OpenSearch-Dashboards/plugins/security-analytics-dashboards-plugin/opensearch_dashboards.json)
```

**Review by phestecAMZN — COMMENTED** [review]
Assuming the OSD version issue works out and tests pass, tests look very thorough.  I'll approve once the tests pass. I'm still a bit new to  cypress, so definitely learned some things here.

File: cypress/integration/3_alerts.spec.js [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+      force: true,
+    });
+    cy.get('[data-test-subj="superDatePickerCommonlyUsed_Today"]').click({ force: true });
+  });
+
+  it('are generated', () => {
+    // Ingest documents to the test index
+    const docCount = 4;
+    for (let i = 0; i < docCount; i++) {
+      cy.insertDocumentToIndex(testIndex, '', sample_document);
```
Comment (amsiglan) [inline cypress/integration/3_alerts.spec.js:105]: Don't these tests run in parallel? If they do, then should this ingestion and waiting for alerts happen before any of the remaining test runs?

Comment (AWSHurneyt) [reply cypress/integration/3_alerts.spec.js:105]: My understanding is that the `cy` commands run asynchronously, but the `describe` and `it` blocks run sequentially based on the `spec` file structure. The `cy.wait` that occurs on line 109 pauses the remaining test cases as well.

Comment (amsiglan) [reply cypress/integration/3_alerts.spec.js:105]: Checked cypress docs and they are run sequentially, so we are fine here

**Review by amsiglan — APPROVED** [review]

**Review by phestecAMZN — APPROVED** [review]
I'll need to update references to data-test-subj's in findings spec that were changed, nevertheless approved.

**Pull request merged**